source/style-guide/style/names.txt

```diff
@@ -4,10 +4,19 @@
 Names
 =====
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 Don't use real or copyrighted names in examples. For example, don't
-use john.smith@example.com.
+use john.smith@example.com or sherlock.holmes@example.com.
+
+Take care to include names that reflect the diversity of the real world.
+Some example names include:
```

Comment:
Are you using the personas?

https://wiki.corp.mongodb.com/display/DESIGN/MongoDB+Personas

Reply:
I'm not; I don't think that's what personas are for.